Add Zip instal to WSL setup in Contributing.md

<!--
When filing an issue, please fill in the requested fields as thoroughly as you can and then delete this comment
Note: For issues, assistance, and questions about using NI products, please visit ni.com/support. Issues opened here should be specifically related to these Python APIs.
-->

### Description of issue

When building on WSL, following the current contributing.md instructions, I ran into this issue when running tox -e codegen:

> Warning: "VoltageLimitRange" not found in function metadata. Typo? Generated code will be funky!
Warning: "OverrangingEnabled" not found in function metadata. Typo? Generated code will be funky!
     Generating: docs/nidcpower/toc.inc
     Generating: docs/nidcpower/errors.rst
     Generating: docs/nidcpower/rep_caps.rst
       Creating: generated/nidcpower/README.rst
     Generating: generated/nidcpower/setup.py
     Generating: generated/nidcpower/tox-system_tests.ini
        Zipping: generated/examples/nidcpower_examples.zip
/bin/sh: 1: zip: not found
make[1]: *** [/home/msms-user/nimi-python/build/rules.mak:78: /home/msms-user/nimi-python/generated/examples/nidcpower_examples.zip] Error 1
make: *** [/home/msms-user/nimi-python/build/Makefile:52: nidcpower] Error 2
ERROR: InvocationError for command /usr/bin/make (exited with code 2)
__________________________________________________________________________________________ summary __________________________________________________________________________________________
ERROR:   codegen: commands failed

This occurs because the `zip` is not installed to on the version of linux Windows setup. Running `sudo apt-get install zip` fixes this.
____________________________________


### Steps to reproduce issue

    1. Follow instructions to setup WSL.
    2. Clone Repository
    3. run tox -e codegen. 
    4. See error.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -35,6 +35,7 @@ following installed:
     - If you're using Windows Subsystem for Linux
         - Install and enable [Windows Subsystem for Linux](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide)
         - Install make: ``sudo apt-get install make``
+        - Install zip: ``sudo apt-get install zip``
 
     - If you're on Windows 10 (not using WSL):
         - [Install mingw (msys-base)](https://osdn.net/projects/mingw/).

